portfolio/src/main/webapp/index.html

```diff
@@ -1,19 +1,83 @@
+        Since I was kid, i was told that going into the medical field would gurantee success.
+        However, it wasn't my main interest. It was technology that would captrue my attention.
+        Whether it was gaming or helping my family with their devices by troubleshooting and finding
+        the issue, it was always my niche.
+    </p1>
+
+    <div>
+        <p1 style="color:white"> I am now attending the California State University of Northridge and am pursuing a
+        degree in Computer Information Technology and a minor in Geographic Information Science.
+        In the future with my degree, I hope to become a Data Scientist. Another occupation that 
+        I am also highly interested in becoming is a Cyber Security Analyst.
+        </p1>
+    </div>
+
+    
+    <div class="w3-panel w3-light-gray w3-round-xlarge">
+        <p4>My LinkedIn and GitHub</p4>
+    </div>
+    <div><p1 style="color:white"> https://github.com/mike39608/Coding-Portfolio </p1></div>
+    <div><p1 style="color:white"> https://www.linkedin.com/in/michael-peters-4686b1227/ </p1></div>
+    
+        <div class="w3-panel w3-light-gray w3-round-xlarge">
+        <p4>My Interests</p4>
+        </div>
+        <p1 style="color:white"> When it comes to what I like, I love to play sports and video games.
+            As well as watching shows, movies and anime. Here are some I am a big fan of.
+            I am currently learning to cook becuase the hobby had always interested me. watching
+            the food channnel and social media accounts propelled my interest into the world of
+            learning to cook foods. 
+            Otherhobbies and skills I would love to learn and practice woudl be learning a new language
+            such as spanish or playing the piano.
+        </p1>
+        <div><img src="/images/under night.jpg" width="400" height="300"><img src="/images/fire force.jpg" width="400" height="300"></div>
+        
+       
+    
+        <div class="w3-panel w3-light-gray w3-round-xlarge">
+        <p4>My projects</p4>
+        </div>
+        <p1 style="color:white"> I have created projects such as a Binary Tree and 
+            and Information System Projects that involve utilizing Access and Excel
+            to analyze statistical data. The list of my work on GitHub. However, especially
+            still being at an entry level, I haven't created very many projects since the 
+            Computer IT major does not focus on creating programs, rather you are handling 
+            pre-built systems and operating systems overall. But I do hope to create more,
+            especially with this opportunity of the summer!
+        </p1>
+        
+        <div class="w3-panel w3-light-gray w3-round-xlarge">
+            <p4>Fun Fact About Me</p4>
+        </div>
+            <p1 style="color:white">The Game that I like the most is Star Wars Battlefront 2</p>
```

Comment:
(There are other cases where this happens in this file I think). 

The number within a tag has semantic meaning. Generally, the smaller the number the more visually important a component is. For instance, titles tend to use `<h1>` while subtitles use `<h3>`.  Of course, you could modify the font size of the tags to make `<p4>` bigger than `<p1>`, but this is discouraged.